optimum/habana/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -415,29 +415,30 @@ def gaudi_gpt_neox_rotary_embedding_set_cos_sin_cache(self, seq_len, device, dty
     self.sin_cached = emb.sin()
 
 
-def apply_customized_rope(q, k, cos, sin, position_ids):
+def apply_customized_rope(q, k, cos, sin, position_ids, training=True):
     if q.device.type == "hpu" and FusedRoPE:
-        # TODO: Enable FusedRoPE with bf16 after accuracy is fixed
-        #     if q.dtype == torch.bfloat16:
-        #         rope_q = FusedRoPE.apply(
-        #             q,
-        #             cos.unsqueeze(0).unsqueeze(0).to(torch.bfloat16),
-        #             sin.unsqueeze(0).unsqueeze(0).to(torch.bfloat16),
-        #             position_ids,
-        #         )
-        #     else:
-        #         rope_q = FusedRoPE.apply(q, cos.unsqueeze(0).unsqueeze(0), sin.unsqueeze(0).unsqueeze(0), position_ids)
-        #     if k.dtype == torch.bfloat16:
-        #         rope_k = FusedRoPE.apply(
-        #             k,
-        #             cos.unsqueeze(0).unsqueeze(0).to(torch.bfloat16),
-        #             sin.unsqueeze(0).unsqueeze(0).to(torch.bfloat16),
-        #             position_ids,
-        #         )
-        #     else:
-        #         rope_k = FusedRoPE.apply(k, cos.unsqueeze(0).unsqueeze(0), sin.unsqueeze(0).unsqueeze(0), position_ids)
-        rope_q = FusedRoPE.apply(q, cos.unsqueeze(0).unsqueeze(0), sin.unsqueeze(0).unsqueeze(0), position_ids)
-        rope_k = FusedRoPE.apply(k, cos.unsqueeze(0).unsqueeze(0), sin.unsqueeze(0).unsqueeze(0), position_ids)
+        if training:  # TODO: Enable FusedRoPE with bf16 after accuracy is fixed
```

Comment:
You may remove #TODO since this is to use the fix for accuracy issue

Reply:
Done.